docusaurus/docs/test-a-plugin/getting-started/target-multiple-grafana-versions.md

```diff
@@ -0,0 +1,157 @@
+---
+id: target-multiple-grafana-versions
+title: Target multiple Grafana versions
+description: Target multiple Grafana versions
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - testing
+  - e2e
+  - ci
+sidebar_position: 3
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Introduction
+
+`@grafana/plugin-e2e` is tested with all minor versions of Grafana since 8.3.0. This article provides examples on how to run e2e tests targeting different Grafana versions locally and in CI.
+
+## On your local machine
+
+In the [installation](./installation.md#start-grafana) guide you saw how to start Grafana by running `npm run server`. When running this command, the latest Grafana Enterprise image will be used. If you want to target a different version of Grafana in your e2e tests, you can use the `GRAFANA_VERSION` environment variable to specify a version of choice.
+
+<Tabs defaultValue="npm">
+<TabItem value="npm">
+
+```bash
+GRAFANA_VERSION=10.1.6 npm run server
+```
+
+</TabItem>
+
+<TabItem value="yarn">
+
+```bash
+GRAFANA_VERSION=10.1.6 yarn server
+```
+
+</TabItem>
+
+<TabItem value="pnpm">
+
+```bash
+GRAFANA_VERSION=10.1.6 pnpm server
+```
+
+</TabItem>
+</Tabs>
+
+## CI
+
+The following workflow can be used to run e2e tests against a matrix of Grafana versions for every PR in your Github repository. Note that this is a generic example based on a backend plugin. You may want to alter or remove a few of the steps in the `playwright-tests` job before using it in your plugin.
+
+<Tabs defaultValue="npm">
+<TabItem value="npm">
+
+```yaml
```

Comment:
This example is using yarn. Working on a few improvements to the `grafana/plugin-actions/publish-report` Action, so would like to complete that before I create copies for npm and pnpm.

Reply:
@sympatheticmoose did you want the workflow do be added to 
[set-up-github-workflows](https://grafana.com/developers/plugin-tools/create-a-plugin/develop-a-plugin/set-up-github-workflows) instead?

Reply:
> Yes I think it makes more sense to keep everything together.

Are we sure about this? All the other workflows in set-up-github-workflows.md can copied and used as is, whereas this workflow is only useful if you've gone through all the other steps in this getting started guide. Feels like it becomes bit of an orphan in that context. 

>I also would have expected the guidance to include changing the versions to test against?

My plan was to also create a longer guide about CI that walks through all Actions and steps in this workflow in depth. Similar to how it's done in Playwright's [getting started](https://playwright.dev/docs/ci-intro) docs and [CI guide](https://playwright.dev/docs/ci#ci-configurations). Thoughts?